### PR TITLE
Everything is finished according to instructions 

### PR DESCRIPTION
**Pull request by edlingao**

### CODE REVIEW

File: css/main.css [hunk 1]
```diff
@@ -0,0 +1,210 @@
+*{
+    padding: 0;
+    margin: 0;
+}
+
+
+.container{
+    display: grid;
+    grid-template-columns: repeat(1, 1rf);
```
Comment (zlayabekrija) [inline css/main.css]: should be fr

File: css/main.css [hunk 2]
```diff
@@ -0,0 +1,210 @@
...
+    grid-template-rows: repeat(3, auto);
```
Comment (zlayabekrija) [inline css/main.css]: auto can't be assigned, repeat expects value

File: css/main.css [hunk 3]
```diff
@@ -0,0 +1,210 @@
...
+    align-items: center;
+    background: #6A6A6A;
+    color: white;
+}
+
+
+.person-week{
+    display: grid;
+    grid-template-columns: repeat(2, 50%);
+    grid-auto-rows: repeat(4,auto);
```
Comment (zlayabekrija) [inline css/main.css]: same here auto shouldn't be repeat

**Review by zlayabekrija — CHANGES_REQUESTED** [review]
Great work, very clean code 🥇 . Yet minor changes needed.
Please follow them up.
Resubmit review request and happy coding :)

File: css/main.css (hunk 2 above)
Comment (zlayabekrija) [reply css/main.css]: in this case, can be omitted since it doesn't have any function.

File: css/main.css (hunk 3 above)
Comment (zlayabekrija) [reply css/main.css]: same here can be omitted, doesn't have any function.

Comment (edlingao) [reply css/main.css]: The change has been committed and everything is fixed

**Comment by edlingao** [comment]
The change has been committed and everything is fixed

**Review by zlayabekrija — APPROVED** [review]
Nice and clean, impressive 🥇 . Thumbs up 👍 , and good look with next projects.
Happy coding 😄 .

**Pull request merged**